README.md

```diff
@@ -18,50 +18,82 @@
 ## Requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8 or later, 64-bit
```

Comment:
thats not true as such since Java 12, 13 , 14.. is not really supported/tested

maybe 

* Java 11, 64-bit (> 11.0.2) 

Not sure to be honest ..  wdyt @electrum ?

Reply:
Yes, maybe leave this alone. I think that people have successfully run under JDK 14, but I know for certain that JDK 15 does not work.